src/ThrottlingMiddleware.php

```diff
@@ -100,7 +100,7 @@ protected function buildException(string $key, int $limit, bool $headers)
 
         $exception = new ThrottlingException($after, 'Rate limit exceeded.');
 
-        $headers = $this->getHeaders($key, $limit, $headers, $after, $exception->getHeaders());
+        $headers = $this->getHeaders($key, (int) $limit, $headers, $after, $exception->getHeaders());
```

Comment:
Limit is already an integer, since it came in via an argument to buildException. That cast won't do anything?

Reply:
Pretty sure when I was testing this, I had to cast it for it to work.

Reply:
You're right, I made too many changes. Will push again.